docs-2.0/3.ngql-guide/6.functions-and-expressions/7.count.md

```diff
@@ -0,0 +1,96 @@
+# count()
+
+The `count()` function calculates the number of the specified values or rows.
+
+## Group and count the number of values
+
+You can use `count(*)` and `GROUP BY` together to group and count the number of specific values.
+
+### Example 1
+
+```nGQL
+nebula> GO FROM "player101" OVER follow BIDIRECT YIELD $$.player.name as Name | \
+        GROUP BY $-.Name YIELD $-.Name, count(*);
++---------------------+----------+
+| $-.Name             | COUNT(*) |
++---------------------+----------+
+| "Dejounte Murray"   | 1        |
++---------------------+----------+
+| "LaMarcus Aldridge" | 2        |
++---------------------+----------+
+| "Tim Duncan"        | 2        |
++---------------------+----------+
+| "Marco Belinelli"   | 1        |
++---------------------+----------+
+| "Manu Ginobili"     | 1        |
++---------------------+----------+
+| "Boris Diaw"        | 1        |
++---------------------+----------+
+Got 6 rows (time spent 4056/4684 us)
+```
+
+The statement in the preceding example searches for:
+
+* People whom `player101` follows.
+* People who follow `player101`.
+
+And retrieves two columns:
+
+* `$-.Name`, the names of the people.
+* `COUNT(*)`, how many times the names show up.
+
+Because there are no duplicate names in the `nba` dataset, the number `2` in the result shows that the person in that row and `player101` have followed each other.
+
+### Example 2
+
+```nGQL
+nebula> MATCH (v:player) RETURN v.age as Age | \
+        GROUP BY $-.Age YIELD $-.Age AS Age, count(*) AS Number | \
+        ORDER BY Number DESC;
++-----+--------+
+| Age | Number |
++-----+--------+
+| 30  | 4      |
++-----+--------+
+| 34  | 4      |
++-----+--------+
+| 33  | 4      |
++-----+--------+
+| 29  | 4      |
++-----+--------+
+| 31  | 3      |
++-----+--------+
+...
+Got 25 rows (time spent 2697/3343 us)
+```
+
+The statement in the preceding example retrieves the age distribution of the players in the dataset.
+
+## Count the rows in the result
+
+You can use `count(*)` in a `YIELD` clause behind a pipe symbol to count the number of rows that the statement before the pipe symbol returns.
```

Comment:
Please reorganize this sentence I think it's a bit confusing. What about
You can use `count(*)` in a `YIELD` clause behind a pipe.  `count(*)` counts the number of rows returned by the statement before the pipe.